evap/results/templates/results_index.html

```diff
@@ -1,22 +1,84 @@
 {% extends 'base.html' %}
 
+{% load results_templatetags %}
+{% load evaluation_filters %}
+
 {% block title %}{% trans 'Results' %} - {{ block.super }}{% endblock %}
 
 {% block content %}
     {{ block.super }}
+    <h3>{% trans 'Evaluation results' %}</h3>
 
-    <h3>{% trans 'Semesters' %}</h3>
-    <div class="card mb-3">
+    <div class="card">
         <div class="card-body">
-            {% if semesters %}
-                <ul>
-                    {% for semester in semesters %}
-                        <li><a href="{% url 'results:semester_detail' semester.id %}">{{ semester.name }}</a></li>
+            {% if courses %}
+                <table class="table table-hover-evap course-table vertically-aligned{% if not forloop.last %} mb-3{% endif %}">
+                    <thead>
+                        <tr>
+                            <th style="width: 35%">{% trans 'Course' %}</th>
+                            <th style="width: 10%">{% trans 'Semester' %}</th>
+                            <th style="width: 22%">{% trans 'Responsible' %}</th>
+                            <th style="width: 13%">{% trans 'Voters' %}</th>
+                            <th style="width: 20%">{% trans 'Distribution and average grade' %}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    {% for course in courses %}
+                        <tr class="results-hover {% if course.is_single_result or not course|can_user_see_results_page:request.user %}nohover{% endif %}{% if course.state == 'in_evaluation' or course.state == 'evaluated' or course.state == 'reviewed' %} preview{% endif %}"
+                        {% if not course.is_single_result and course|can_user_see_results_page:request.user %}data-url="{% url 'results:course_detail' course.semester.id course.id %}"{% endif %}>
```

Comment:
fix indentation please, also the condition can be put into a variable

Reply:
everything inside the for loop was moved to a different file, but i can still fix it in this commit.

do you mean putting `course.is_single_result or not course|can_user_see_results_page:request.user` into a variable? `or` is unfortunately not allowed in `with`, so it would have to be two variables. or did you mean something else?